frontend/src/lib/components/WarningModal.svelte

```diff
@@ -0,0 +1,20 @@
+<svelte:options runes={true} />
+
+<script lang="ts">
+import { Button, Modal } from "flowbite-svelte";
+import ExclamationCircleOutline from "flowbite-svelte-icons/ExclamationCircleOutline.svelte";
+import { _ } from "svelte-i18n";
+
+let { open = $bindable(false), text }: { open: boolean; text: string } =
+	$props();
+</script>
+
+<Modal bind:open size="xs" autoclose>
```

Comment:
would it be possible to make this such that it leaves the parent modal open? currently it seems that,e.g., when I upload an image that is too large in the milestonegroup it throws me back to the milestonegroup overview and I have to open the edit element for the milestonegroup again to try another image.

Reply:
good catch! I guess I never clicked the Verstanden button, as if you click on the x in the top right of the modal this doesn't happen.
And yes I think this PR is the right place to fix this issue - but we can and should do some usability testing at some point looking for minor niggles like this

Reply:
should be fixed by 4440830